src/pages/contacts-page/contacts-page.js

```diff
@@ -74,28 +74,24 @@ const ContactsPage = () => {
     );
   }, [dispatch, contactsPerPage, contactsCurrentPage]);
 
-  const changePageHandler = (e, pageIndex) =>
+  const changePageHandler = (e, pageIndex) => {
     dispatch(setContactsCurrentPage(pageIndex));
-
-  const contactItems =
-    contacts !== undefined
-      ? contacts.map((contact, index) => (
-        <TableContainerRow
-          key={index}
-          id={contact.id}
-          phone={contact.phoneNumber}
-          email={contact.email}
-          address={contact.address[0].value.replace(
-            formRegExp.unwrapHtml,
-            ' '
-          )}
-          deleteHandler={() => contactDeleteHandler(contact._id)}
-          editHandler={() => {
-            dispatch(push(`/contacts/${contact._id}`));
-          }}
-        />
-      ))
-      : null;
+  };
+  const contactItems = contacts
+    ? contacts.map((contact, index) => (
+      <TableContainerRow
+        key={index}
```

Comment:
Don't use `index` as a key, use some unique value from the `contact` object, like `id`